docs/orchestrate/building-images.md

```diff
@@ -8,8 +8,8 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-
 ## Overview
+
 Sauce Labs supports Docker images, providing you with a powerful way to orchestrate tests. In order to use Hosted Orchestration you will need to package your test code and all of its dependencies as a Docker image and publish it to a Docker container registry so that your tests can be run in the Sauce Labs infrastructure.
```

Comment:
Sauce Labs supports container runtime, such as Docker or containerd, providing you with a powerful way to orchestrate tests.